app/views/employment/SingleEmploymentSummaryView.scala.html

```diff
@@ -77,7 +77,7 @@
 </div>
 }
 
-@layout(pageTitle = headingForUse, taxYear = Some(taxYear), isAgent = user.isAgent) {
+@layout(pageTitle = messages("employment.employmentSummary.heading"), taxYear = Some(taxYear), isAgent = user.isAgent) {
```

Comment:
Has the content changed from what was demoed last week with this page?

Reply:
Yeah Joey pointed out that the title was different if it was a single employment or multiple so made them the same title, but keeping the heading as is. 

Having the title as the employer name is likely to be a personal information issue as they will be seen on google analytics.